docs/CHANGELOG.asciidoc

```diff
@@ -28,7 +28,12 @@
 
 //=== Regressions
 
-== {es} version 7.17.9
+== {es} version 7.17.11
+
+=== Enhancements
+
+* Better handle small shifts of the seasonal patterns in time series data.
+  (See {ml-pull}2202[#2202].)
 
```

Comment:
```suggestion

== {es} version 7.17.9

```

Reply:
@tveasey please can you include this change too. As it stands you've moved a 7.17.9 bug fix to 7.17.13.